### PR TITLE
Fixes #17352: improve documentation on "default" for variable to state that it should be quoted

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/17352

### CODE REVIEW

File: src/reference/modules/reference/pages/variables.adoc [hunk 1]
```diff
@@ -1116,17 +1116,19 @@ This is also a good case to allow a simple override mechanism for a parameter wh
...
+
 Some examples:
 
 ----
 
 ${node.properties[datacenter][id] | default = "LON2" }
 ${node.properties[datacenter][name] | default = """Co-location with "Hosting Company" in Paris (allows quotes)""" }
-${node.properties[datacenter][id] | default = ${rudder.parameters[default_datacenter]} }
-${node.properties[netbios_name] | default = ${rudder.node.hostname} }
+${node.properties[datacenter][id] | default = "${rudder.parameters[default_datacenter]} }"
```
Comment (ncharles) [inline src/reference/modules/reference/pages/variables.adoc:1130]: the closing quote should not be there

Comment (ncharles) [inline src/reference/modules/reference/pages/variables.adoc:1130]:
```suggestion
${node.properties[datacenter][id] | default = "${rudder.parameters[default_datacenter]}" }
```

File: src/reference/modules/reference/pages/variables.adoc [hunk 2]
```diff
@@ -1116,17 +1116,19 @@ This is also a good case to allow a simple override mechanism for a parameter wh
...
+${node.properties[netbios_name] | default = "${rudder.node.hostname} }"
```
Comment (ncharles) [inline src/reference/modules/reference/pages/variables.adoc:1131]:
```suggestion
${node.properties[netbios_name] | default = "${rudder.node.hostname}" }
```

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**